### ISSUE TITLE
Issue Notes should be less ephemeral

### ISSUE THREAD

**Issue by monkeysecurity** [Netflix/security_monkey#475]
Typical flow:

- Find item with issue.
- Issue uses policy statement as a note.  
- Justify issue.
- Policy statement changes.  
- Issue seen as "fixed" and replaced with a new unjustified issue.

Super annoying to lose justifications in this way.

Should probably stop dumping policy statements into the notes field.  It may be important to review all the auditors and consider changing them.

Alternatively, the ItemAudit table could have a new "ephemeral_notes" column.  The display of the issue would be the concatenation of the current `notes` column with the new `ephemeral_notes` column.  The `ephemeral_notes` column would not be used when determining whether an issue currently exists, so things like policy statements could be placed there.

I'll leave this issue open for a few days.  Please leave your thoughts or suggested fixes.

**Comment by monkeysecurity** [comment]
Example issue:
```
Issue: IAM Role has IAM privileges.	
Score: 9
Notes: {"Action": [...,iam:something,...]}
```

I'm going to annoyingly tag a few people for their thoughts now :)  No rush.

@ollytheninja @cstewart87 @aebie @markofu @bunjiboys @crruthe @mikegrima

**Comment by markofu** [comment]
@monkeysecurity this approach with the "ephemeral_notes" column seems reasonable to me.

**Comment by aebie** [comment]
@monkeysecurity I agree with @markofu that ephemeral_notes is reasonable, but I am looking at some "IAM Role has IAM privileges." issues and some of the actions are really lengthy and clutter the screen. It seems redundant to list the action too since the config is already right there. I think maybe this is not a proper use of the notes field.

**Comment by cstewart87** [comment]
I agree with removing the policy dumps from the notes field.  However, I'm struggling with why you would add a new "ephemeral_notes" column instead of just making the existing "notes" column ephemeral.  What's the requirement for using "notes" to determine whether an issue exists?

**Comment by monkeysecurity** [comment]
Fixed in #564.
